docs/retrieverjl.rst

```diff
@@ -0,0 +1,131 @@
+==========================
+Data Retriever using Julia
+==========================
+
+The wrapper module for `Data Retriever <http://data-retriever.org>`_ has been implemented as `Retriever.jl <https://github.com/weecology/retriever.jl.git>`_. All the functions work and feel the same as the python Retriever module. The module has been created using ``PyCall`` hence all the functions are analogous to the functions of retriever python module.
+
+
+Installation
+============
+
+To use Julia package ``retriever.jl``, you first need to `install Retriever <introduction.html#installing-binaries>`_.
+
```

Comment:
Installation
============

To use Retriever, you first need to `install Retriever  <introduction.html#installing-binaries>`_. a core python package

To  install ``Retriever`` using the Julia package manager

```julia
julia> Pkg.add("Retriever")
```


To install from Source,
download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.

Install  ``PyCall`` package.

.. code-block:: julia

    """ Julia Command """

    Pkg.install("PyCall")


Go to ``retriever.jl/src``. Run Julia.

.. code-block:: julia
    
    """ Julia Command """
    
    include("retriever.jl")


(Note: If you want help in installing Julia you can follow this `tutorial <https://medium.com/@shivamnegi2019/julia-beginners-guide-part-1-a9c369128c78>`_)


.. code-block:: julia

    """ Julia Command """

    Pkg.install("PyCall")

To install from Source, download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.

Go to ``retriever.jl/src``. Run Julia.

.. code-block:: julia
    
    """ Julia Command """
    
    include("retriever.jl")


(Note: If you want help in installing Julia you can follow this `tutorial <https://medium.com/@shivamnegi2019/julia-beginners-guide-part-1-a9c369128c78>`_)